openbb_sdk/pyproject.toml

```diff
@@ -1,34 +1,36 @@
 [tool.poetry]
 name = "openbb-sdk"
-version = "4.0.0"
-description = ""
+version = "4.0.0rc9"
+description = "OpenBB SDK"
 authors = ["OpenBB Team <hello@openbb.co>"]
 readme = "README.md"
 packages = [{ include = "openbb" }]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<3.12"
-openbb-provider = { path = "./sdk/provider" }
-openbb-core = { path = "./sdk/core" }
-openbb-stocks = { path = "./extensions/stocks" }
-openbb-crypto = { path = "./extensions/crypto" }
-openbb-forex = { path = "./extensions/forex" }
-openbb-futures = { path = "./extensions/futures" }
-openbb-news = { path = "./extensions/news" }
-openbb-economy = { path = "./extensions/economy" }
-openbb-fixedincome = { path = "./extensions/fixedincome" }
-openbb-charting = { path = "./extensions/charting"}
-openbb-benzinga = { path = "./providers/benzinga" }
-openbb-fmp = { path = "./providers/fmp" }
-openbb-polygon = { path = "./providers/polygon" }
-openbb-fred = { path = "./providers/fred" }
-openbb-yfinance = { path = "./providers/yfinance" }
-openbb-cboe = { path = "./providers/cboe"}
+openbb-core = {version = "^0.0.0a7", source = "test-pypi"}
+openbb-benzinga = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-fmp = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-fred = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-polygon = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-crypto = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-economy = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-fixedincome = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-forex = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-news = {version = "^0.0.0a3", source = "test-pypi"}
+openbb-qa = {version = "^0.0.0a3", source = "test-pypi"}
```

Comment:
ta & qa should also be either removed or moved to extras like we have forecasting and portfolio in v3